SPEC.md

```diff
@@ -216,11 +216,31 @@ Additional isolators will be added to this specification over time.
 |network-io/write-bandwidth |string|"&lt;device name&gt; &lt;bytes&gt;" |"eth0 100M"                         |
 |capabilities/bounding-set  |string|"&lt;cap&gt; &lt;cap&gt; ..."       |"CAP_NET_BIND_SERVICE CAP_SYS_ADMIN"|
 
-#### Types
+##### Types
 
 * uint: base 10 formatted unsigned int as a string
 * bytes: Suffix to a base 10 int to make it a K, M, G, or T for base 1024
 
+#### Devices and File Systems
+
+Apps should expect to run in an environment with basic device nodes and filesystems.
+The exact device nodes and filesystems available depend on the operating system.
+
+##### Linux
+
+|     Path     |  Type  |
+| ------------ | ------ |
+| /proc        | proc   |
+| /dev/null    | devfs  |
+| /dev/zero    | devfs  |
+| /dev/full    | devfs  |
+| /dev/random  | devfs  |
+| /dev/urandom | devfs  |
+| /dev/tty     | devfs  |
+| /dev/net/tun | devfs  |
```

Comment:
My vote is for `/dev/net/tun` not be part of the default set. It's only used by specialized software, may not exist on the host (e.g. kernel compiled without it). I'm actually not sure if /dev/net/tun is universal path for it. Of course we need to have a way for an app to request it explicitly.